orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/config/SelectableServiceSpec.groovy

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.config
+
+import com.netflix.spinnaker.orca.clouddriver.KatoService
+import com.netflix.spinnaker.orca.clouddriver.MortService
+import com.netflix.spinnaker.orca.clouddriver.OortService
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll;
+
+class SelectableServiceSpec extends Specification {
+  @Shared
+  def mortService = Mock(MortService)
+
+  @Shared
+  def oortService = Mock(OortService)
+
+  @Shared
+  def katoService = Mock(KatoService)
+
+  @Unroll
+  def "should lookup service by application or executionType"() {
+    given:
+    def selectableService = new SelectableService(
+      [
+        new ByApplicationServiceSelector(mortService, ["applicationPattern": ".*spindemo.*"]),
+        new ByExecutionTypeServiceSelector(oortService, ["executionTypes": [0: "orchestration"]]),
```

Comment:
is the [0: "orchestration"] really how the yaml deserializes?

I'd expect
```
config:
  executionTypes:
    - orchestration
```
to deserialize to [config: executionTypes: ['orchestration']]`